ert/serialization/_registry.py

```diff
@@ -3,12 +3,18 @@
 from pyrsistent import pmap
 from pyrsistent.typing import PMap
 
-from ._serializer import Serializer, _json_serializer, _yaml_serializer
+from ._serializer import (
+    Serializer,
+    _ecl_sum_serializer,
+    _json_serializer,
+    _yaml_serializer,
+)
 
 _registry: PMap[str, Serializer] = pmap(
     {
         "application/json": _json_serializer(),
         "application/x-yaml": _yaml_serializer(),
+        "application/eclsum": _ecl_sum_serializer(),
```

Comment:
Is this created now? In that case, if the type represents multiple components that might have their own media types, `multipart` is the correct type. Also, if there are multiple subtypes, maybe `multipart/x-eclipse.sum` an `eclipse` _tree_ might be more fitting (or something).

Reply:
I agree. `multipart/x-eclipse.sum` seems appropriate, `multipart` because two files are always needed (for summary data), and `.sum` since `multipart/x-eclipse` should probably refer to the whole set of output files from E100/E300.

Reply:
I like the idea of subtypes; ie. `multipart/x-eclipse.sum`, just here comes the ambiguity as `EclSum` handles both files and folders.